src/routeApp/validations.js

```diff
@@ -0,0 +1,21 @@
+import { body } from 'express-validator';
+
+const routeValidation = () => [
+	// body('code', 'Code is required').notEmpty().toUpperCase(),
+	body('origin', 'Origin is required ').notEmpty(),
+	body('destination', 'Destination is required ').notEmpty(),
+	body('distance', 'This should not be empty').optional().isLength({
+		min: 3,
+	}),
```

Comment:
will the distance be optional on the backend?

Reply:
nop, but it is no biggy on our backend side coz it will be calculated on the frontend side. thanks mn